Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/UX - Microsoft Consumer Experience Disabled.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "Turns off Microsoft consumer specific experiences, such as Start suggestions, membership notifications, post-OOBE app install and redirect tiles.",
+  "description": "What is it: Turns off Microsoft consumer specific experiences, such as start suggestions, membership notifications, post-OOBE app install and redirect tiles. \n Simeon Notes: Improves the user enrollment experience. \n Learn More: https://docs.microsoft.com/en-us/windows/client-management/mdm/policy-csp-experience#experience-allowwindowsconsumerfeatures",
```

Comment:
**What does this do?**
Disables Microsoft Consumer Experiences such as Start suggestions, Membership notifications, Post-OOBE app install and redirect tiles.

**Why should you use this?**
If you want to improve the user experience by eliminating non value-add notifications and suggestions. 

**What is the end user impact?**
Users will not see these additional pop-ups, suggestions and notifications.  

**Learn more**
[Policy CSP - Experience](https://docs.microsoft.com/en-us/windows/client-management/mdm/policy-csp-experience#experience-allowwindowsconsumerfeatures)